src/services/doppler-legacy-client.ts

```diff
@@ -313,6 +340,7 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
   }
 
   public async login(model: LoginModel): Promise<LoginResult> {
+    this.setRetryAxiosAndLog();
```

Comment:
I think that it is not a good solution because the _Axios_ instance is shared.

Maybe you could have more than an _Axios_ instance?

For example:

```js
  constructor({ axiosStatic, baseUrl }: { axiosStatic: AxiosStatic; baseUrl: string }) {
    this.baseUrl = baseUrl;
    this.axios = axiosStatic.create({
      baseURL: baseUrl,
      withCredentials: true,
    });
    this.axiosRetry = axiosRetry(axiosStatic.create({
      baseURL: baseUrl,
      withCredentials: true,
    }), {
    retries: 3,
    shouldResetTimeout: true,
    retryCondition: () => {
      return true;
    },
    retryDelay: (retryCount: any, error: any) => {
      addLogEntry({
        origin: window.location.origin,
        section: 'Login/legacyClient',
        browser: window.navigator.userAgent,
        error: error,
        retryCount: retryCount,
      });
      return retryCount * 1000;
    },
  });
}
```

Reply:
Another option could be using the same axios instance but modify _retryCondition_ to only work for login.

Reply:
Good point!